test/parallel/test-eventtarget.js

```diff
@@ -162,6 +162,27 @@ let asyncTest = Promise.resolve();
   eventTarget.dispatchEvent(ev);
 }
 
+{
+  // Same event dispatched multiple times.
+  const event = new Event('foo');
+  const eventTarget1 = new EventTarget();
+  const eventTarget2 = new EventTarget();
+
+  eventTarget1.addEventListener('foo', common.mustCall((event) => {
+    strictEqual(event.target, eventTarget1);
+  }));
+
+  eventTarget2.addEventListener('foo', common.mustCall((event) => {
+    strictEqual(event.target, eventTarget2);
+  }));
+
+  eventTarget1.dispatchEvent(event);
+  strictEqual(event.target, null);
```

Comment:
On the browser (tested on Chromium, Firefox and Safari), `event.target` is not null:

```js
{
  // Same event dispatched multiple times.
  const event = new Event('foo');
  const eventTarget1 = new EventTarget();
  const eventTarget2 = new EventTarget();

  eventTarget1.addEventListener('foo', ((event) => {
    console.log(1, event.target === eventTarget1);
  }));

  eventTarget2.addEventListener('foo', ((event) => {
    console.log(2, event.target=== eventTarget2);
  }));

  eventTarget1.dispatchEvent(event);
  console.log(event.target ===null, event.target ===eventTarget1); // false true

  eventTarget2.dispatchEvent(event);
  console.log(event.target ===null, event.target===eventTarget2); // false true
}
```
I don't know if that should be blocking, but it'd be nicer to have a consistent behavior across platforms.

Reply:
Noticed, but it is also not `undefined`.

Reply:
I think that it should eventually be addressed in a different commit/PR.